src/parser/data/Variable.h

```diff
@@ -65,5 +74,8 @@ class Variable {
   }
 
   // ___________________________________________________________________________
-  [[nodiscard]] std::string toString() const { return _name; }
+  [[nodiscard]] std::string toSparql() const { return _name; }

```

Comment:
This can also be a `const string&` ,
If other types return `std::string` here, no harm is done imho.

Reply:
This code is supposed to be removed sooner or later, so I'd prefer to keep it as-is for now, even if this might avoid an unecessary copy